<launch>

<!-- MoveBaseSeq node launch and Points Sequence ROS Parameters Loading -->

    <!--Run move_base_seqs node-->
        <node name="multi_goal_navigation" pkg="multi_goal_nav" type="multi_goal_script.py"  output="screen">
        
        <!-- Load Sequence of Points (Cartesian Coordinates wrt "map" RF, in meters) -->
            <rosparam param="p_left_seq">[-1.25,0,0,-1.25,0,0,-1.25,1.70,0,-1.25,1.70,0]</rosparam>
        <!-- Load Sequence of Points (Cartesian Coordinates wrt "map" RF, in meters) -->
            <rosparam param="p_right_seq">[-1.25,0,0,-0.75,1.40,0,-0.45,1.70,0]</rosparam>
        <!-- Load Sequence of Points (Cartesian Coordinates wrt "map" RF, in meters) -->
            <rosparam param="p_start_seq">[-0.85,0,0,-0.28,0,0]</rosparam>
        <!-- Load Sequence of Desired Yaw Angles (no rotations around x and y axes) in degrees-->
            <rosparam param="yea_start_seq">[0,0]</rosparam>
        <!-- Load Sequence of Desired Yaw Angles (no rotations around x and y axes) in degrees-->
            <rosparam param="yea_left_seq">[30,90,90,90]</rosparam>
        <!-- Load Sequence of Desired Yaw Angles (no rotations around x and y axes) in degrees-->
            <rosparam param="yea_right_seq">[0,50,70]</rosparam>

        </node>

</launch>




